products/views.py

```diff
@@ -4,15 +4,17 @@
 from django.http import HttpResponse
```

Comment:
`Http404` is a symbol from `django.http`, not `django.shortcuts`

```suggestion
from django.http import Http404, HttpResponse
```